Makefile

```diff
@@ -127,7 +127,7 @@ gas: $(GAS)
 # convenience targets
 all: components tethers isos vic-machine vic-ui
 tools: $(GOIMPORTS) $(GVT) $(GOLINT) $(SWAGGER) $(GAS) goversion
-check: goversion goimports govet golint copyright whitespace gas
+check: goversion goimports govet lintcode copyright whitespace gas
```

Comment:
btw, target golint doesn't work for some reason. I am not an expert in Makefile(s), I would be really happy if somebody gives me a valuable advice why it is not working.